ci-operator/step-registry/openshift-qe/installer/bm/day2/netperf-external/openshift-qe-installer-bm-day2-netperf-external-commands.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+# Setup Bastion
+  ## Change default gateway to Bastion host
+  ## Host Netserver at $EXTERNAL_SERVER_ADDRESS
+
+SSH_ARGS="-i ${CLUSTER_PROFILE_DIR}/jh_priv_ssh_key -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
+bastion=$(cat ${CLUSTER_PROFILE_DIR}/address)
+
+ssh ${SSH_ARGS} root@"${bastion}" bash -s <<EOF
```

Comment:
To fix the below error https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/68230/pull-ci-openshift-release-master-step-registry-shellcheck/1974148389276749824 

Add below line just before the ssh
`#` shellcheck disable=SC2087
ssh ${SSH_ARGS} root@"${bastion}" bash -s <<EOF

For example https://github.com/openshift/release/blob/master/ci-operator/step-registry/network/mtu-migration/network-mtu-migration-commands.sh#L31